### PR TITLE
Add a test case for private nested classes.

### LINKED ISSUE #155: private inner classes should not be emitted

### PR DESCRIPTION
**Pull request by mprobst**
Does not fix #155.

### CODE REVIEW

File: src/test/java/com/google/javascript/clutz/privates.js [hunk 1]
```diff
@@ -23,3 +23,6 @@ priv.PublicClass.prototype.publicField;
 
 /** @private @return {number} */
 priv.PublicClass.prototype.method_ = function() { return this.field; };
+
+/** @constructor @private */
+priv.PublicClass.PrivateNestedClass_ = function() {};
```
Comment (rkirov) [inline src/test/java/com/google/javascript/clutz/privates.js]: `goog.provide('priv.PublicClass');` to see the bug

Comment (mprobst) [inline src/test/java/com/google/javascript/clutz/privates.js]: Fixed.

**Comment by mprobst** [comment]
Note that this is now based on #157.

**Comment by rkirov** [comment]
LGTM

**Pull request merged**